src/store/modules/AccountsModule.js

```diff
@@ -41,15 +49,56 @@ class AccountsModule extends VuexModule {
     return !!this.settings.otpEnabled;
   }
 
+  get ethBalance() {
+    if (!this.isBalanceLoading && this.balance) {
+      return fromWei(this.balance);
+    }
+    return '0';
+  }
+
+  /**
+   * @param {object} newSettings
+   */
+  @Action
+  async changeSettings(newSettings) {
+    const oldSettings = this.settings;
+    this.settings = newSettings;
+
+    const isNetworkChanged = oldSettings.net !== newSettings.net;
+    if (isNetworkChanged) {
+      setWeb3Network(newSettings.net);
+    }
+
+    this.resubscribeBalance({ isNetworkChanged, newSettings, oldSettings });
```

Comment:
Leave `TODO` here about resubscribing to `eth` prices in future, when related logic will appear here